langchain4j-agentic/src/main/java/dev/langchain4j/agentic/observability/AgentListenerProvider.java

```diff
@@ -0,0 +1,8 @@
+package dev.langchain4j.agentic.observability;
+
+import dev.langchain4j.Internal;
+
+@Internal
+public interface AgentListenerProvider {
```

Comment:
```suggestion
public interface AgenticListenerProvider {
```

or perhaps `AgenticListener` should be renamed into `AgentListener` for consistency

Reply:
Done.